regression/contracts/variant_parsing_multiple_clauses_fail/test.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+main.c
+--apply-loop-contracts
+activate-multi-line-match
```

Comment:
Why do we need this here?

Reply:
It's necessary for the multi-line regex match for parsing error (see the next line).